packages/cli/src/util/env/get-env-records.ts

```diff
@@ -49,3 +49,41 @@ export default async function getEnvRecords(
 
   return client.fetch<{ envs: ProjectEnvVariable[] }>(url);
 }
+
+export async function pullEnvRecords(
+  output: Output,
+  client: Client,
+  projectId: string,
+  source: EnvRecordsSource,
+  {
+    target,
+    gitBranch,
+  }: {
+    target?: ProjectEnvTarget | string;
+    gitBranch?: string;
+  } = {}
+) {
+  output.debug(
+    `Fetching Environment Variables of project ${projectId} and target ${target}`
+  );
+  const query = new URLSearchParams();
+
+  let url = `/v1/env/pull/${projectId}`;
+
+  if (target) {
+    const gitBranchPath = gitBranch ? `/${gitBranch}` : '';
+    url += `/${target}${gitBranchPath}`;
+  }
```

Comment:
How are you ensuring that `gitBranch` or `target` don't already have a `/` character at the front of them? I'd rather see something like `path.join` be used or utilize the `URL` class